extra/migrate-fa-v4.php

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env php
+<?php
+// @codingStandardsIgnoreFile
+
+/**
+ * Helper script to migrate icons used in WoltLab Suite before version 6.0.
+ *
+ * @author Alexander Ebert
+ * @copyright 2001-2022 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core
+ */
+
+if (\PHP_SAPI !== 'cli') {
+    exit;
+}
+
+// JSON Structure:
+// [key: string]: {
+//   name: string;
+//   type: "brand" | "regular" | "solid";
+// }
+$json = <<<JSON
```

Comment:
```suggestion
$json = <<<'JSON'
```
To be extra sure, that `$` won't be interpreted in there.